build/README.md

```diff
@@ -5,3 +5,43 @@ To build the Rollups Node Docker image, run the following command in the build d
 ```
 docker buildx bake --load
 ```
+
+# Run Node Devnet infrastrucuture
+
+## Generate Devnet state
+
+First you need to generate Anvil state executing the following commands in a shell 
+```
+cd anvil
+docker compose up
+```
+
+Wait until you see a log like the one bellow and stop execution 
+```
+anvil-hardhat-1 exited with code 0
+```
+
+## Run infrastructure with sample echo DApp
+
+After the base Anvil state files are generated (see section above), at `build` folder, run :
+```
+docker compose up
+```
+
+## Send input
+
+```
+INPUT=0x68656C6C6F206E6F6465 ;\
+INPUT_BOX_ADDRESS=0x59b22D57D4f067708AB0c00552767405926dc768 ;\
+DAPP_ADDRESS=0x70ac08179605AF2D9e75782b8DEcDD3c22aA4D0C ;\
+cast send $INPUT_BOX_ADDRESS "addInput(address,bytes)(bytes32)" $DAPP_ADDRESS $INPUT --mnemonic "test test test test test test test test test test test junk" --rpc-url "http://localhost:8545"
+```
+
+## query input
+
+Access `http://localhost:4000/graphql` and execute query
+```
+query{
+  notices {edges{node{index,payload,input{payload}}}}
+}
```

Comment:
You can export a curl example in the GraphQL. It would be clearer instead of asking the person to open the browser.